broker-daemon/utils/exponential-backoff.js

```diff
@@ -0,0 +1,21 @@
+async function exponentialBackoff (callFunction, attempts, delay, callback) {
+  try {
+    var res = await callFunction()
+  } catch (e) {
+    console.log(e)
+  }
+
+  if (res) {
+    callback(res)
+  } else {
+    if (attempts > 0) {
+      setTimeout(function () {
```

Comment:
There are timer limits, but im not sure if we would ever hit them (its like 24 days or something). With that said, do we want to limit the amount of attempts?